tests/filecheck/dialects/arith/arith_attrs.mlir

```diff
@@ -0,0 +1,27 @@
+// RUN: xdsl-opt %s -t mlir | xdsl-opt -f mlir -t mlir | filecheck %s
```

Comment:
Do we need the `-t mlir` here? Or is this already the default?

Reply:
I see that it's not yet the default. I guess we first switch all the tests and then change the default.